Linux/external-data-mirror/ansible/readme.md

```diff
@@ -0,0 +1,75 @@
+# External Data Mirror Deployment
+
+This allows for additional copies of the external data to be generated to reduce the load
+on the main server. 
+
+Developers deploying these playbooks will require ssh access to `mantidproject.org`.
+
+## Setup
+
+### VM Provisioning
+- Create a new linux virtual machine (VM) (on OpenStack at STFC) with an ssh key that 
+  you have on your system.
+- Add the `HTTP` security group to the VM to allow traffic into the node on port `80`.
+
+### Ansible Setup
+- Create an `ansible` conda environment:
+
+```sh
+mamba create -n ansible ansible
+mamba activate ansible
+```
+
+- Install 3rd party ansible packages from `ansible-galaxy`:
+
+```sh
+ansible-galaxy install -r requirements.yml
+```
+
+- Create an `inventory.txt` file in the following format:
+
+    - `VM_IP_ADDRESS`: IP address of the node you just provisioned. 
+    - `IP_TO_COPY_FROM`: IP address or domain (usually `mantidproject.org`) that holds 
+      the external data you want to copy.
+    - `DIR_NAME`: The directory in the `/srv/` directory on the server that holds the 
+      rest of the path to the external data. On `mantidproject.org` this is formatted 
+      as the main server's IP.
+
+```ini
+[all]
+<VM_IP_ADDRESS> main_server_hostname=<IP_TO_COPY_FROM> main_data_srv_dir=<DIR_NAME>
+```
```

Comment:
In addition, a parameter called `external_data_location` needs to be added to the inventory file. It would be better to add info about it